definitions/procedures/restore/candlepin_dump.rb

```diff
@@ -0,0 +1,32 @@
+module Procedures::Restore
+  class CandlepinDump < ForemanMaintain::Procedure
+    metadata do
+      description 'Restore candlepin postgresql dump from backup'
+      param :backup_dir,
+            'Path to backup directory',
+            :required => true
+      preparation_steps { Checks::Candlepin::DBUp.new unless feature(:candlepin_database).local? }
+      confine do
+        feature(:candlepin_database)
+      end
+    end
+
+    def run
+      backup = ForemanMaintain::Utils::Backup.new(@backup_dir)
+
+      with_spinner('Restoring candlepin postgresql dump') do |spinner|
+        feature(:service).handle_services(spinner, 'start', :only => ['postgresql'])
```

Comment:
this does not work on remote database, because postgresql service is not installed at all, getting error
```
E, [2018-05-09 11:00:00+0200 #32464] ERROR -- : No services found matching your parameters (RuntimeError)
/usr/share/gems/gems/foreman_maintain-0.2.1/definitions/features/service.rb:44:in `filtered_services'
/usr/share/gems/gems/foreman_maintain-0.2.1/definitions/features/service.rb:15:in `handle_services'
/usr/share/gems/gems/foreman_maintain-0.2.1/definitions/procedures/restore/candlepin_dump.rb:18:in `block in run'
```

the same thing fails also in `definitions/procedures/restore/foreman_dump.rb`
mongo works `definitions/procedures/restore/mongo_dump.rb` but I would fix it too to prevent failures one mongod was removed

Reply:
The goal is to fix f-m service to handle remote DBs correctly. It should not restart db services for remote dbs and only run db ping there for `start` and `status`. I'll prepare PR soon.